core/src/main/java/com/linecorp/armeria/client/circuitbreaker/CircuitBreakerRpcClientBuilder.java

```diff
@@ -35,7 +36,16 @@
      * Returns a newly-created {@link CircuitBreakerRpcClient} based on the properties of this builder.
      */
     public CircuitBreakerRpcClient build(RpcClient delegate) {
-        return new CircuitBreakerRpcClient(delegate, mapping(), ruleWithContent());
+        return build(delegate, CircuitBreakerClientHandler.of(CircuitBreakerMapping.ofDefault()));
```

Comment:
`AbstractCircuitBreakerClientBuilder` also has `CircuitBreakerClientHandler.of(CircuitBreakerMapping.ofDefault())`.
How about adding `CircuitBreakerClientHandler.of()` and using it?

Reply:
`CircuitBreakerClientHandler` is generic so it's a little difficult to statically pre-instantiate the handler. Instead, I removed the duplicate instantiation 🙏